FashionConnect_api/routes/users.js

```diff
@@ -71,25 +71,29 @@ router.post('/users/login', async (req, res) => {
 });
 
 
+
 //routes for profile changes
+
 router.post('/users/profile', async (req, res) => {
     const { First_Name, Last_Name, username } = req.body;
   
     try {
       // Find the user by username
       const user = await User.findOne({ where: { username:username } });
-      console.log(user)
-  
-      if (!user) {
-        return res.status(404).json({ error: 'User not found' });
+
+      if (!user){
+        res.status(404).json({error: "User does not exist"});
       }
 
       user.First_Name = First_Name
       user.Last_Name = Last_Name
+      console.log(user)
       await user.save()
-  
-      // Return the user data in the response
-      res.json({ message: "User profile uodated succesfully" });
+     
+
+    console.log(user)
+    res.json({ message:"User info updated", user: user});
```

Comment:
Try to maintain the same indent for your code. :)

Reply:
Yes i will.